website/docs/r/vapp_vm.html.markdown

```diff
@@ -243,25 +243,15 @@ The following arguments are supported:
 * `cpus` - (Optional) The number of virtual CPUs to allocate to the VM. Socket count is a result of: virtual logical processors/cores per socket. The default is 1. If `cpu_hot_add_enabled` is true, then cpus will be increased without VM power off.
 * `cpu_cores` - (Optional; *v2.1+*) The number of cores per socket. The default is 1
 * `metadata` - (Optional; *v2.2+*) Key value map of metadata to assign to this VM
-* `initscript` (Optional **Deprecated** by `customization.0.initscript`) Script to run on initial boot or with
-customization.force=true set. See [Customization](#customization-block) to read more about Guest customization and other
-options.
 * `storage_profile` (Optional; *v2.6+*) Storage profile to override the default one
-* `network_name` - (Optional; **Deprecated** by `network`) Name of the network this VM should connect to.
-* `vapp_network_name` - (Optional; v2.1+; **Deprecated** by `network`) Name of the vApp network this VM should connect to.
-* `ip` - (Optional; **Deprecated** by `network`) The IP to assign to this vApp. Must be an IP address or
-one of `dhcp`, `allocated`, or `none`. If given the address must be within the
-  `static_ip_pool` set for the network. If left blank, and the network has
-  `dhcp_pool` set with at least one available IP then this will be set with
-DHCP.
 * `power_on` - (Optional) A boolean value stating if this VM should be powered on. Default is `true`
 * `accept_all_eulas` - (Optional; *v2.0+*) Automatically accept EULA if OVA has it. Default is `true`
```

Comment:
When I run my configuration on this PR it results in an unexpected change:

```
  # vcd_vapp.demo_vapp will be updated in-place
  ~ resource "vcd_vapp" "demo_vapp" {
      + accept_all_eulas = true
        guest_properties = {}
        href             = "https://bos1-vcloud-static-172-8.eng.vmware.com/api/vApp/vapp-7752b19f-52e2-4fae-8a61-5297319907dd"
        id               = "urn:vcloud:vapp:7752b19f-52e2-4fae-8a61-5297319907dd"
        ip               = "allocated"
        metadata         = {}
        name             = "demo-web"
        power_on         = true
        status           = 4
        status_text      = "POWERED_ON"
    }
```

This is my config:

```
# vApp
resource "vcd_vapp" "demo_vapp" {
  name = "demo-web"

  power_on = true

  depends_on = ["vcd_network_routed.demo_routed_net_web"]
}
```

Reply:
will be removed in https://github.com/vmware/terraform-provider-vcd/pull/558

Reply:
Looks like I got this, because was using build from #563, which removed the field, and then switched to this branch, which doesn't have the removal yet and wants to add it.